src/pages/home/Carousel.tsx

```diff
@@ -0,0 +1,77 @@
+import { Link } from "react-router-dom";
+
+function Carousel() {
+  return (
+    <div
+      id="carouselExampleDark"
+      className="carousel carousel-dark slide"
+      data-bs-ride="carousel"
+    >
+      <div className="carousel-indicators">
+        <button
+          type="button"
+          data-bs-target="#carouselExampleDark"
+          data-bs-slide-to="0"
+          className="active"
+          aria-current="true"
+          aria-label="Slide 1"
```

Comment:
If you have an hour sometime this week I think it would be worthwhile to try pulling in the bootstrap react component here and wiring it up.